src/components/postCard/postCard.js

```diff
@@ -108,7 +116,7 @@ export const PostCard = ({ data, authId }) => {
         setLikeCount((pre) => (pre === 1 ? 0 : -1));
       }
     } catch (error) {
-      console.log(error);
+      // console.log(error);
```

Comment:
maybe add toast at this place? to show user 'the error'

Reply:
i will add in next pr